index.html

```diff
@@ -117,12 +124,27 @@ <h2>OS metrics reporter</h2>
                             </a>
                         </div>
                         <div class="col-xs-12 col-md-6 col-lg-4 project-box">
-                            <a href="https://github.com/smartcat-labs/smartcat-guidelines">
-                                <time>May 15, 2016</time>
-                                <h2>SmartCat guidelines</h2>
-                                <p>Coding and contributing guidelines used by SmartCat.</p>
+                            <a href="https://github.com/smartcat-labs/twitalyzr">
+                                <time>Jan 15, 2017</time>
+                                <h2>Twitalyzr</h2>
+                                <p>Detection of valuable and non-valuable tweats based on Cassandra hash tag.</p>
+                            </a>
+                        </div>
+                        <div class="col-xs-12 col-md-6 col-lg-4 project-box">
+                            <a href="https://github.com/smartcat-labs/smartcat-ops">
+                                <time>May 10, 2017</time>
+                                <h2>Neural Blog Writter</h2>
+                                <p>Use neural network to train the model which can write blog post based on curren blog posts on some website. </p>

```

Comment:
Use neural network to train a model which can generate blog posts based on posts published so far.